### PR TITLE
Added operating system switch

### PR DESCRIPTION
**Pull request by SuNNjek**

### CODE REVIEW

File: DepotDownloader/Program.cs [hunk 1]
```diff
@@ -94,7 +94,8 @@ static void Main( string[] args )
             ContentDownloader.Config.MaxServers = GetParameter<int>( args, "-max-servers", 20 );
             ContentDownloader.Config.MaxDownloads = GetParameter<int>( args, "-max-downloads", 4 );
             string branch = GetParameter<string>( args, "-branch" ) ?? GetParameter<string>( args, "-beta" ) ?? "Public";
-            var forceDepot = HasParameter( args, "-force-depot" );
+            bool forceDepot = HasParameter( args, "-force-depot" );
+            string os = GetParameter<string>( args, "-os", null );
```
Comment (yaakov-h) [inline DepotDownloader/Program.cs]: As @psychonic said in IRC, this should probably throw an error or something if someone does both `-os` and `-all-platforms`

File: DepotDownloader/Program.cs [hunk 2]
```diff
@@ -166,6 +167,7 @@ static void PrintUsage()
             Console.WriteLine( "\t-password <pass>\t\t\t- the password of the account to login to for restricted content." );
             Console.WriteLine( "\t-remember-password\t\t\t- if set, remember the password for subsequent logins of this user." );
             Console.WriteLine( "\t-dir <installdir>\t\t\t- the directory in which to place downloaded files." );
+            Console.WriteLine( "\t-os <os>\t\t\t- the operating system for which to download the game (windows, macos or linux, default: OS the programm is currently running on)" );
```
Comment (yaakov-h) [inline DepotDownloader/Program.cs]: Typo: "programm" => "program"

File: README.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ Optional Parameters:
   -password <pass>       - the password of the account to login to for restricted content.
   -remember-password     - if set, remember the password for subsequent logins of this user.
   -dir <installdir>      - the directory in which to place downloaded files.
+  -os <os>               - the operating system for which to download the game (windows, macos or linux, default: OS the programm is currently running on)
```
Comment (yaakov-h) [inline README.md]: Same typo

File: DepotDownloader/Program.cs (hunk 1 above)
Comment (SuNNjek) [reply DepotDownloader/Program.cs]: I added an error message now (and fixed the typos). I hope the message is clear enough.

**Pull request merged**